src/pages/workspaces/workspace/analysis/modals/CloudEnvironmentModal.js

```diff
@@ -451,6 +471,7 @@ export const CloudEnvironmentModal = ({
     const isCloudEnvForToolDisabled = isCloudEnvModalDisabled(toolLabel);
     return h(Fragment, [
       // We cannot attach the periodic cookie setter until we have a running Cromwell app for Azure because the relay is not guaranteed to be ready until then
+      // TODO: this assumes the presence of a Cromwell app. Make more generic to key off presence of any Azure app.
```

Comment:
Making note to fix this -- `PeriodicAzureCookieSetter` should kick off of there is _any_ azure app; not just Cromwell specifically.

Reply:
fixed